assets/app/view/game/part/track_node_path.rb

```diff
@@ -278,6 +279,11 @@ def load_from_tile
                                  ]
                                end
             end
+            lanes = if @path.a.edge?
+                      @path.ab_lanes
+                    else
+                      [@path.ab_lanes.last, @path.ab_lanes.first]
```

Comment:
could just be 

lanes = @path.ab_lanes
lanes.reverse! if @path.a.edge?

Reply:
same with all the other ones

Reply:
Nice. Will do.